### ISSUE TITLE
Create a style guide

### ISSUE THREAD

**Issue by rgreinho** [request-yo-racks/web#8]
This application does not have a style guide whatsoever. It would be great to be able to propose colors, logos, icons, etc. in order to help the developers who have limited design knowledge to build something pretty.

An example of style guide I like is https://frontify.com/styleguide.

**Comment by rgreinho** [comment]
Some style guides I found pretty cool:

- [Gitbook](http://styleguide.gitbook.com/)
- [Mapbox](https://www.mapbox.com/base/)

**Comment by rgreinho** [comment]
[codeforamerica](http://codeforamerica.clearleft.com/)

**Comment by rgreinho** [comment]
Even more examples: [styleguides.io](http://styleguides.io/examples)

**Comment by kcwoodfield** [comment]
Not sure if this is a solved problem but are you familiar with fabricator? 

https://github.com/fbrctr/fabricator

**Comment by rgreinho** [comment]
@kcwoodfield I did not know about this project, But it looks exactly what we need! However, it has been archived, and it looks like it is abandoned :( Is there any active alternative?

**Comment by kcwoodfield** [comment]
@rgreinho you're probably right.  I wonder if that is a recent thing.  Anyhow, lets not use this then - it may create more headaches than we want.

**Comment by rgreinho** [comment]
@kcwoodfield This page seems to have very interesting tools to create style guides: https://github.com/davidhund/styleguide-generators

**Comment by rgreinho** [comment]
I really like this one: http://astrum.nodividestudio.com/pattern-library/#group-branding-component-icons

**Comment by kcwoodfield** [comment]
@rgreinho I really like that one.  

If using react this is nice also: https://github.com/storybooks/storybook

**Comment by rgreinho** [comment]
@kcwoodfield It's definitely a nice tool, but it is to help people developing UI components, not creating a living styleguide.

**Comment by rgreinho** [comment]
@UX-Chris Since we merged #34 , can we close this issue? Or do you still need to work more on the styleguide?

**Comment by UX-Chris** [comment]
No, a lot of the style has changed so I'll update the style guide as get get things finished.
